apps/web-mzima-client/src/app/post/post-details/post-details.component.scss

```diff
@@ -83,6 +83,9 @@
       margin-bottom: 0;
     }
   }
+  &__metadata-container {
+    margin-block: 2rem;
```

Comment:
We don't use rem units within the application, so you can just stick with the px or em. Let the value here be or be equivalent to 16px for consistency @mannyAndem

Reply:
Will fix